### PR TITLE
Updated styleguide with test guide

### PR DESCRIPTION
**Pull request by DenFlyvendeGed**
### Changes/Additions:

- Added unit test folder to the  section folder structure of the styleguide
- Changed to folder structure of the end-to-end test folder
- Modified the console unit tests

### CODE REVIEW

File: styleguide.md [hunk 1]
```diff
@@ -161,13 +162,35 @@ The following folder structure is meant to be a guideline of where one should pu
...
-            Folder structure should copy the project structure,
-            and put tests in their corresponding folders -->
+    └── <-- End-to-end test files here.
+            Folder structure should copy the project structure starting from src,
+            and put tests in their corresponing folders -->
+```
+
+### Tests writing guidelines:
+
+- There should be a one to one relation between files testet and test files.
```
Comment (atjn) [inline styleguide.md:180]:
```suggestion
- There should be a one to one relation between files tested and test files.
```

Comment (TSKsmiley) [inline styleguide.md:180]: tested
```suggestion
- There should be a one to one relation between files testet and test files.
+ There should be a one to one relation between files tested and test files.
```

**Review by TSKsmiley — DISMISSED** [review]
sounds nice to me

**Review by KamyaPA — APPROVED** [review]
LGTM

**Review by TSKsmiley — APPROVED** [review]

**Pull request merged**